src/xSTIR/pSTIR/STIR.py

```diff
@@ -470,6 +478,23 @@ def set_modality(self, mod):
         """
         return parms.set_char_par(self.handle, 'ImageData', 'modality', mod)
 
+    @property
+    def __array_interface__(self):
+        """As per https://numpy.org/doc/stable/reference/arrays.interface.html"""
+        if not self.contiguous:
+            raise ContiguousError("please make a copy first with `as_array`")
```

Comment:
I wonder if this text isn't a bit confusing. `as_array()` doesn't really make a copy of the `ImageData` (or whatever) object. I suggest
```suggestion
            raise ContiguousError("please make a array-copy first with `as_array`")
```